app/models/session.rb

```diff
@@ -245,6 +245,11 @@ def close!
         on_duplicate_key_ignore: true
       )
 
+      Consent
```

Comment:
Is it worth being extra careful and using `policy_scope(Consent)` here?

Reply:
Hmm yes, that's a good point.